core/sql/parser/sqlparser.y

```diff
@@ -780,6 +781,7 @@ static void enableMakeQuotedStringISO88591Mechanism()
 %token <tokval> TOK_LAST_SYSKEY
 %token <tokval> TOK_LASTSYSKEY
 %token <tokval> TOK_LCASE               /* ODBC extension   */
+%token <tokval> TOK_LEAD                /* LEAD OLAP Function */
```

Comment:
It is better to add TOK_LEAD and TOK_LAG into the nonreserved_func_word production as well.